shoplist/src/main/java/com/netguru/codereview/network/ShopListRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network
+
+class ShopListRepository(private val shopListApi: ShopListApi) {
```

Comment:
Recommended to create ShopListRepository interface and implement it in ShopListRepositoryImpl, it will ensure to have good Dependency Inversion control.

interface  ShopListRepository {
//all the methods
}

class ShopListRepositoryImpl : ShopListRepository{
//ovverdidden methods
}